cmd/ssh_usercert_gen/main.go

```diff
@@ -430,7 +430,10 @@ func (state *RuntimeState) checkAuth(w http.ResponseWriter, r *http.Request) (st
 func (state *RuntimeState) SaveUserProfiles() error {
 	var gobBuffer bytes.Buffer
 	encoder := gob.NewEncoder(&gobBuffer)
-	encoder.Encode(state.userProfile)
+	err := encoder.Encode(state.userProfile)
+	if err != nil {
```

Comment:
You can put this in one line:
if err := encoder.Encode(state.userProfile); err != nil {